cpanfile

```diff
@@ -50,3 +50,5 @@ requires 'Protocol::Matrix', '>= 0.02';
 requires 'Struct::Dumb', '>= 0.04';
 requires 'URI::Escape';
 requires 'YAML';
+
+requires 'URI::Escape::XS';
```

Comment:
this file is sorted... can you stick it in the right place please

Reply:
Is there a reason to use this instead of the URI::Escape we already have? Sytest already downloads most of the internet :/

Reply:
(plus if you bump the deps you have to rebuild the docker images and faff)

Reply:
Oh, I got confused with `URI::Encode`, which says to use `URI::Escape:XS`